crates/bitwarden-exporters/src/lib.rs

```diff
@@ -132,3 +134,12 @@ pub struct Identity {
     pub passport_number: Option<String>,
     pub license_number: Option<String>,
 }
+
+pub struct SshKey {
+    /// Ssh private key (ed25519/rsa) in unencrypted openssh private key format https://github.com/openssh/openssh-portable/blob/master/PROTOCOL.key
+    pub private_key: Option<String>,
+    /// Ssh public key (ed25519/rsa) according to RFC4253 https://datatracker.ietf.org/doc/html/rfc4253#section-6.6
+    pub public_key: Option<String>,
+    /// Ssh fingerprint using SHA256 in the format: SHA256:BASE64_ENCODED_FINGERPRINT
```

Comment:
Rust comments supports markdown. While we don't use `UPPER CASE` format in code, we should use it in comments for abbreviations.

```suggestion
    /// SSH fingerprint using SHA256 in the format: `SHA256:BASE64_ENCODED_FINGERPRINT`
```